curriculum/challenges/english/25-front-end-development/lab-survey-form/66a8e9ca371965b6ef5f2989.md

```diff
@@ -0,0 +1,600 @@
+---
+id: 66a8e9ca371965b6ef5f2989
+title: Build a Survey Form
+challengeType: 14
+dashedName: build-a-survey-form
```

Comment:
```suggestion
dashedName: lab-build-a-survey-form
```

Reply:
Looks like the challenges just stick with the `build-a-survey-form` structure.